packages/terra-tag/src/Tag.jsx

```diff
@@ -59,13 +64,31 @@ class Tag extends React.Component {
     // Apply focus styles for keyboard navigation
     if (event.nativeEvent.keyCode === KeyCode.KEY_TAB) {
       this.setState({ focused: true });
+    } else if (event.nativeEvent.keyCode === KeyCode.KEY_RIGHT) {
+      this.setState({ focused: true });
+    } else if (event.nativeEvent.keyCode === KeyCode.KEY_LEFT) {
+      this.setState({ focused: true });
+    } else if (event.nativeEvent.keyCode === KeyCode.KEY_RETURN) {
+      this.setState({ focused: true });
     }
```

Comment:
These seem like a bunch of redundant checks for the same outcome. We can optimize this using a single `if` block and a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/Set).

Reply:
changes done. Thanks